README.md

```diff
@@ -41,6 +44,99 @@ This plugin can be consumed by the CAP application deployed on BTP to store thei
 >
 > A reference to adding this can be found [here](https://github.com/cap-java/sdm/blob/691c329f4c3c17ae390cfcb2db1ef02650585aee/cap-notebook/demoapp/pom.xml#L20)
 
+## Setup
+
+In this guide, we use the Bookshop reference sample app in the [deploy branch](https://github.com/cap-java/sdm/tree/deploy) of this repository, to integrate SDM CAP plugin.
+
+### Using the released version
+If you want to use the released version of SDM CAP plugin follow the below steps:
+
+1. Delete the sdm and sdm-root folders from your local .m2 repository
+
+2. Clone the sdm repository:
+
+```sh
+   git clone https://github.com/cap-java/sdm
+```
+
+3. Checkout to the branch **deploy**:
+
+```sh
+   git checkout deploy
+```
+
+4. Navigate to the demoapp folder:
+
+```sh
+   cd cap-notebook/demoapp
+```
+
+5. Modify the [xsappname](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/xs-security.json#L1) value in the **xs-security.json**
+
+6. Modify the [xsappname](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L82), [REPOSITORY_ID](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L21) and SDM instance name in the [srv module](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L31) and the [resources section](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L98) values in the **mta.yaml**
```

Comment:
Same as above. What should these values be modified to?

Reply:
Changed